app/controllers/admin/bug_tickets_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::BugTicketsController < AdminController
+  def index
+    @bugs = BugTicket.includes(:user).page(params[:page]).reverse_order.decorate
```

Comment:
`BugTicket.includes(:user).page(params[:page]).reverse_order`
move to `bugs` method
and this order.
`BugTicket.includes(:user).reverse_orde.page(params[:page])`

Then in a action

`@bugs = bugs.decorate`